dev-python/text-unidecode/text-unidecode-1.3.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="The most basic Text::Unidecode port"
+HOMEPAGE="https://github.com/kmike/text-unidecode/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
I'm not sure about this file https://github.com/kmike/text-unidecode/blob/a0005d7a0b43a6aa21ce8a63f1da48c6c44f03e1/src/text_unidecode/data.bin

but I still hope it using for decoding unicode chars :)
https://github.com/kmike/text-unidecode/blob/a0005d7a0b43a6aa21ce8a63f1da48c6c44f03e1/src/text_unidecode/__init__.py#L6-L21